src/docker/composebuilder/development.go

```diff
@@ -149,14 +147,6 @@ func (d *DevelopmentDockerComposeBuilder) getServiceFilePath() string {
 	return path.Join(d.AppDir, d.ServiceData.Location)
 }
 
-func (d *DevelopmentDockerComposeBuilder) getDockerLinks() []string {
-	result := []string{}
-	for _, dependency := range d.ServiceConfig.Development.Dependencies {
-		result = append(result, fmt.Sprintf("%s%s:%s", dependency.Name, dependency.Version, dependency.Name))
-	}
-	return result
```

Comment:
I don't believe this is needed if we switch from put the container name as the env var value instead of the dependency name